core/src/components/modal/modal.ios.scss

```
@@ -87,3 +87,7 @@
 :host(.modal-sheet) .modal-wrapper {
   @include border-radius(var(--border-radius), var(--border-radius), 0, 0);
 }
+
+:host(.modal-sheet.modal-scroll-all) ion-footer ion-toolbar.ios:first-of-type {
+  padding-top: 6px;
+}
```

```suggestion
// iOS Sheet Modal - Scroll at all breakpoints
// --------------------------------------------------

/*
 * Sheet modals require an additional padding as mentioned in the
 * `core.scss` file. However, there's a workaround that requires
 * a cloned footer to be added to the modal. This is only necessary
 * because the core styles are not being applied to the cloned footer.
 * It's important to update the padding value when the core styles are
 * updated.
 */
:host(.modal-sheet.modal-scroll-all) ion-footer ion-toolbar:first-of-type {
  padding-top: 6px;
}
```
No need to include `.ios` since the styles are already within the iOS file. Let's also add the comment.